src/libcore/char/methods.rs

```diff
@@ -547,10 +547,10 @@ impl char {
         }
     }
 
-    /// Returns `true` if this `char` satisfies the 'XID_Start' Unicode property, and false
+    /// Returns `true` if this `char` satisfies the `XID_Start` Unicode property, and false
```

Comment:
Keep

Reply:
I think it's clearer when reading the documentation to highlight this with backticks and it does refer to something which is code-like. At least if not with backticks it would be good to use *'XID_Start'* to highlight more.

Reply:
"Keep" means "keep the change to backticks".